test/e2e/theia_clickhouse_test.go

```diff
@@ -0,0 +1,358 @@
+                   reverseOctetTotalCount,
+                   reversePacketDeltaCount,
+                   reverseOctetDeltaCount,
+                   sourcePodName,
+                   sourcePodNamespace,
+                   sourceNodeName,
+                   destinationPodName,
+                   destinationPodNamespace,
+                   destinationNodeName,
+                   destinationClusterIP,
+                   destinationServicePort,
+                   destinationServicePortName,
+                   ingressNetworkPolicyName,
+                   ingressNetworkPolicyNamespace,
+                   ingressNetworkPolicyRuleName,
+                   ingressNetworkPolicyRuleAction,
+                   ingressNetworkPolicyType,
+                   egressNetworkPolicyName,
+                   egressNetworkPolicyNamespace,
+                   egressNetworkPolicyRuleName,
+                   egressNetworkPolicyRuleAction,
+                   egressNetworkPolicyType,
+                   tcpState,
+                   flowType,
+                   sourcePodLabels,
+                   destinationPodLabels,
+                   throughput,
+                   reverseThroughput,
+                   throughputFromSourceNode,
+                   throughputFromDestinationNode,
+                   reverseThroughputFromSourceNode,
+                   reverseThroughputFromDestinationNode)
+                   VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?,
+                           ?, ?, ?, ?, ?, ?, ?, ?, ?, ?,
+                           ?, ?, ?, ?, ?, ?, ?, ?, ?, ?,
+                           ?, ?, ?, ?, ?, ?, ?, ?, ?, ?,
+                           ?, ?, ?, ?, ?, ?, ?)`
+	recordPerCommit = 1000
+	insertInterval  = 1
+	threshold       = 25
+	MaxInt32        = 1<<31 - 1
+)
+
+var targetTable = map[string]string{
+	".inner.flows_node_view":   "16",
+	".inner.flows_pod_view":    "20",
+	".inner.flows_policy_view": "27",
+	"flows":                    "49",
+}
+
+func TestTheiaStatusCommand(t *testing.T) {
+	data, _, _, err := setupTestForFlowVisibility(t, false, false, false)
+	if err != nil {
+		t.Fatalf("Error when setting up test: %v", err)
+	}
+	defer func() {
+		teardownTest(t, data)
+		teardownFlowVisibility(t, data, false)
+	}()
+
+	clientset := data.clientset
+	kubeconfig, err := data.provider.GetKubeconfigPath()
+	require.NoError(t, err)
+	connect, pf, err := commands.SetupClickHouseConnection(clientset, kubeconfig, "", false)
+	require.NoError(t, err)
+	if pf != nil {
+		defer pf.Stop()
+	}
+
+	t.Run("testTheiaGetClickHouseDiskInfo", func(t *testing.T) {
+		testTheiaGetClickHouseDiskInfo(t, data)
+	})
+	t.Run("testTheiaGetClickHouseTableInfo", func(t *testing.T) {
+		testTheiaGetClickHouseTableInfo(t, data, connect)
+	})
+	t.Run("testTheiaGetClickHouseInsertRate", func(t *testing.T) {
+		testTheiaGetClickHouseInsertRate(t, data, connect)
+	})
+
+}
+
+func testTheiaGetClickHouseDiskInfo(t *testing.T, data *TestData) {
```

Comment:
An example of CLI output for each test function will be helpful.
Same for other test functions.

Reply:
Added.